pages/pipelines/build_exports.md

```diff
@@ -0,0 +1,89 @@
+# Build exports
+
+> 📘 Enterprise feature
+> Build exports is only available on an [Enterprise](https://buildkite.com/pricing) plan, which has a [build retention](/docs/pipelines/build-retention) period of 12 months.
+
+If you need to retain build data beyond the [retention period](/docs/pipelines/build-retention) in your [Buildkite plan](https://buildkite.com/pricing), you can export the data to your own [Amazon S3 bucket](https://aws.amazon.com/s3/).
+
+If you don't configure an S3 bucket, Buildkite stores the build data for 18 months in case you need it. You cannot access this build data through the API or Buildkite dashboard, but you can request the data by contacting support.
+
+## How it works
+
+Builds older than the build retention limit are automatically exported as JSON to the S3 bucket you have configured. If you haven't configured a bucket for build export, Buildkite stores that build data as JSON in our own S3 bucket for a further 18 months in case you need it. The following diagram outlines this process.
+
+<%= image "build-exports-flow-chart.png", alt: "Simplified flow chart of the build exports process" %>
+
+Buildkite exports each build as multiple gzipped JSON files, which include the following data:
+
+```
+buildkite/build-exports/org={UUID}/date={YYYY-MM-DD}/pipeline={UUID}/build={UUID}/
+├── annotations.json.gz
+├── artifacts.json.gz
+├── build.json.gz
+├── step-uploads.json.gz
+└── jobs/
+    ├── job-{UUID}.json.gz
+    └── job-{UUID}.log
+```
+
+The files are stored in the following formats:
+* [Annotations](https://buildkite.com/docs/apis/rest-api/annotations#list-annotations-for-a-build)
+* [Artifacts](https://buildkite.com/docs/apis/rest-api/artifacts#list-artifacts-for-a-build)
+* [Builds](https://buildkite.com/docs/apis/rest-api/builds#get-a-build) (but without `jobs`, as they are stored in separate files)
+* Jobs (as would be embedded in a [Build via the REST API](https://buildkite.com/docs/apis/rest-api/builds#get-a-build))
+
+## Configure build exports
+
```

Comment:
Let's add a sentence here to avoid having the two headings up against each other. Perhaps:

```suggestion

To configure build exports for your organization, you'll need to prepare an S3 bucket before enabling exports in the Buildkite dashboard.

```